runelite-client/src/main/java/net/runelite/client/plugins/blastfurnace/BlastFurnaceCofferOverlay.java

```diff
@@ -77,9 +77,13 @@ public Dimension render(Graphics2D graphics)
 			sack.setHidden(true);
 
 			tableComponent.addRow("Coffer:", StackFormatter.quantityToStackSize(client.getVar(BLAST_FURNACE_COFFER)) + " gp");
```

Comment:
dont forget to add this back to the panelcomponent since the table is getting removed